### PR TITLE
[react-syntax-highlighter] Add missing languages and styles; Up to date for v13.5.3

### PR DESCRIPTION
**Pull request by michaelyuen**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- ~~[ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)~~
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/react-syntax-highlighter/react-syntax-highlighter
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- ~~[ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)~~
- ~~[ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.~~

## Other Notes
- [x] Removed Definition Owner no longer on GitHub

### CODE REVIEW

File: types/react-syntax-highlighter/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 // Type definitions for react-syntax-highlighter 11.0
```
Comment (rbuckton) [inline types/react-syntax-highlighter/index.d.ts:1]: If these changes are specific to v13.5.3, then this line needs to be changed accordingly:
```suggestion
// Type definitions for react-syntax-highlighter 13.5
```

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/react-syntax-highlighter/index.d.ts (hunk 1 above)
Comment (michaelyuen) [reply types/react-syntax-highlighter/index.d.ts:1]: Thank you! I missed this, but also wasn't entirely sure what to do.

Question: Given all the missing definitions I've added, I think it's likely that segments arrived in various versions of the library. How would I handle that? Should the version reflect the _last_ version to support my changes? Assuming the changes arrived at various versions, how do I handle the intermediate versions?

Comment (michaelyuen) [reply types/react-syntax-highlighter/index.d.ts]: I double-checked and the changes I've added do coordinate with v13.5, so I've updated that according to your comment.

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-syntax-highlighter ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-syntax-highlighter@latest/))
was missing the following properties:
1. createElement
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4d7ff66c8982f151dfee49a7383b8554a776210d
</p>

**Comment by yaeda** [comment]
@rbuckton Would you proceed with the review? I want to use this update.

**Comment by orta** [comment]
Cool, this looks good to me 👍🏻

**Pull request merged**